### ISSUE TITLE
Logstash is missing DLQ metrics

### ISSUE THREAD

**Issue by Deroin** [DataDog/integrations-extras#909]
Back on Agent 5.x the logstash integration supplied a metric `logstash.pipeline.dead_letter_queue.queue_size_in_bytes`. With this you could easily monitor whether or not documents get rejected by logstash.

We are now in the process of upgrading to Agent 7.x and noticed, that the integration no longer reports this value. It would be great to get this feature back.

**Output of the [info page](https://docs.datadoghq.com/agent/guide/agent-commands/#service-status)**

```text
===============
Agent (v7.27.0)
===============

  Status date: 2021-06-14 09:01:37.502 UTC (1623661297502)
  Agent start: 2021-06-14 08:39:47.593 UTC (1623659987593)
  Pid: 3.153379e+06
  Go Version: go1.14.12
  Python Version: 3.8.5
  Build arch: amd64
  Agent flavor: agent
  Check Runners: 4
  Log File: /var/log/datadog/agent.log
  Log Level: info

...

    logstash (1.0.0)
    ----------------
      Instance ID: logstash:8bc0d46af7a8cccc [OK]
      Configuration Source: file:/etc/datadog-agent/conf.d/logstash.d/conf.yaml
      Total Runs: 87
      Metric Samples: Last Run: 136, Total: 11,560
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 1, Total: 87
      Average Execution Time : 24ms
      Last Execution Date : 2021-06-14 09:01:27 UTC (1623661287000)
      Last Successful Execution Date : 2021-06-14 09:01:27 UTC (1623661287000)
```

**Comment by hithwen** [comment]
Pinging @ervansetiawan as the logstash integration maintainer.
However if you look at the check history https://github.com/DataDog/integrations-extras/commits/master/logstash/datadog_checks/logstash/logstash.py it does not seem like the functionality has changed since it was developed. Also `logstash.pipeline.dead_letter_queue.queue_size_in_bytes` is not present on t[he list of metrics that integration emits](https://github.com/DataDog/integrations-extras/blob/master/logstash/metadata.csv). 

Maybe @ervansetiawan  can help shed some light here

**Comment by Deroin** [comment]
You're right, and after checking our side of this I noticed that we actually added this with a custom implementation to our 5.x agents. From the looks of it we simply added `"logstash.pipeline.dead_letter_queue.queue_size_in_bytes": ("gauge", "pipeline.dead_letter_queue.queue_size_in_bytes")` to the `PIPELINE_METRICS` hash.

If you want I can create a pull request with this change. If you don't want this to be part of the integration we will try to get a custom version of this in again.
